integrations/sage.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Sage API was added with https://github.com/Bearer/Pizzly/pull/128. Can you rebase your PR? Or remove the `sage.json` file?

Reply:
done